qdisc.go

```diff
@@ -120,6 +125,73 @@ func (qdisc *Htb) Type() string {
 	return "htb"
 }
 
+// Netem is a classless qdisc that rate limits based on tokens
+
+type NetemQdiscAttrs struct {
+	Latency   uint32 // in us
```

Comment:
I am not sure it is worth it given that it will need to be casted back to a uint32 in netlink.